sdk/python/kfp_tekton/tekton.py

```diff
@@ -311,3 +311,39 @@ def __enter__(self) -> Union[Tuple[TektonLoopIterationNumber, LoopArguments], _f
   def enumerate(self) -> dsl.ParallelFor:
     self.call_enumerate = True
     return self
+
+
+class AddOnGroup(dsl.OpsGroup):
+    """
+    Represents an AddOn group containing ops and group of OpsGroups.
+    This class is the base class for a customized OpsGroup. Users can
+    develop their own OpsGroup. The customized OpsGroup maps to a
+    custom task in Tekton.
+    """
+    TYPE_NAME = 'addon_group'
+    TASK_TYPE = 'task'
+    RUN_TYPE = 'run'  # means custom task
+    DEFAULT_KIND = 'AddOnGroup'
+    DEFAULT_APIVERSION = 'custom.tekton.dev/v1alpha1'
+
+    def __init__(self, task_type: str = RUN_TYPE,
+                kind: str = DEFAULT_KIND,
+                api_version: str = DEFAULT_APIVERSION,
+                is_finally: bool = False,
+                parallelism: int = None,
```

Comment:
Doesn't seem to change anything when I set it...

Reply:
do you mean `parallelism`? for `parallelism`, it's from OpsGroup. I added it to be compatible with OpsGroup. In the tekton compiler, I don't have any special handling for it now.